benchcab/data/config-schema.yml

```diff
@@ -29,17 +29,38 @@ realisations:
   schema:
     type: "dict"
     schema:
-      path:
-        type: "string"
+      repo:
+        type: "dict"
+        required: true
+        schema:
+          git:
+            type: "dict"
+            excludes: "svn"
+            schema:
+              branch:
+                type: "string"
+                required: true
+              commit:
+                type: "string"
+                required: false
+              url:
+                type: "string"
+                required: false
```

Comment:
Are you suggesting that we only allow the branch/commit and then the URL is populated internally to point at the CABLE repo?

Reply:
The use case I had in mind was if users want to run benchcab on a fork of CABLE, then they can specify the URL to their forked repo. The URL defaults to the main CABLE git repo.

Reply:
@bschroeter Since this is optional, there is already a default URL that is used internally, so I was wondering if we wanted to allow forks or not. But I guess it does not hurt to allow it so that's fine to keep.